<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>database</class>
 <widget class="QMainWindow" name="database">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>密码管理工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget:focus{outline: none;}
QWidget{
	font: 9pt &quot;微软雅黑&quot;;
		background-color: rgb(255, 255, 255);
}
QScrollBar:vertical
{
    width:0px;
}
QPushButton:hover{
background-color: rgb(112, 162, 255);
}
QPushButton{
background-color: rgb(72, 144, 254);
color: rgb(255, 255, 255);
border-radius:4px;
}

QLineEdit{
border-radius:4px;
border:1px solid #1B89CA;
}

QTableWidget{
border:1px solid #1B89CA;
selection-color: rgb(90, 90, 90);
selection-background-color: rgb(170, 255, 255);
}

</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>911</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>81</y>
        <width>760</width>
        <height>323</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>760</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>760</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>账户</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>密码</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>备注1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>备注2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>上次修改时间</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>10</y>
        <width>30</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/close.png</normaloff>:/close.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>901</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72,144, 254);</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;密 码 管 理 工 具&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>81</y>
        <width>144</width>
        <height>323</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>144</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid #ACACFF;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_AllClear">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>一键清空数据库</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_encrypteFile">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>加密数据库文件</string>
       </property>
      </widget>
     </widget>
     <zorder>tableWidget</zorder>
     <zorder>widget</zorder>
     <zorder>groupBox</zorder>
     <zorder>closeButton</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>901</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(72, 144, 255);</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>160</y>
         <width>261</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;请先解个锁啦：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton_2">
       <property name="geometry">
        <rect>
         <x>870</x>
         <y>0</y>
         <width>30</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/close.png</normaloff>:/close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
